ci/setup-build-test-gdk.ps1

```diff
@@ -166,6 +166,7 @@ Foreach ($test in $tests) {
     # Only run tests on Windows, as we do not have a linux agent - should not matter
     if ($env:BUILD_PLATFORM -eq "Win64" -And $env:BUILD_TARGET -eq "Editor" -And $env:BUILD_STATE -eq "Development") {
         Start-Event "test-gdk" "command"
+        $verify_commandlet_exit_codes = $False # For now, we will ignore the exit codes of the commandlets run, as older engine versions with TestGyms produce errors
```

Comment:
can't we version check this somehow, check if >= 4.25 or whatever version is appropriate?

Reply:
that was my initial idea, but turns out, no, not in general, unless we are willing to add a whole bunch of complexity.